dynamite_flask_app/schemas.py

```diff
@@ -0,0 +1,13 @@
+"""This module creates the schemas required to parse database tables to resources."""
+
+from models import Product, Size, User
+from init_db import ma
+
+
+class ProductSchema(ma.ModelSchema):
+
+    class Meta:
+        model = Product
+
+product_schema = ProductSchema()
```

Comment:
Avoid creating instances here. Create where it is being used

Reply:
Done.